src/main/external-resources/renderers/Onkyo-TXNR8xx.conf

```diff
@@ -24,6 +24,7 @@ UserAgentSearch = Mediabolic-IMHTTP/1.8.225
 UpnpDetailsSearch = ONKYO.*TX-NR8
 LoadingPriority = 1
 
+TranscodeAudio = WAV
 atz_limit = 2000
```

Comment:
@Sami32 That's what I suspected ;)

I can do that manually - but I just discovered something strange. When using Kodi, the FFmpeg transcoded DTS MA files are playable.